readme.md

```diff
@@ -42,8 +42,11 @@ This project is useful when you’re writing documentation for an open source
 project, typically a Node.js package, that has one or more readmes and maybe
 some other markdown files as well.
 You want to show the author and license associated with the project.
+
 When this plugin is used, authors can add a certain heading (say, `## License`)
-to documents and this plugin will populate them.
+to documents and this plugin will populate that section with the information,
+using either the `author` and `license` fields set in a `package.json` file,
+or parameters with the same name passed to the plugin.
```

Comment:
I don't think this is much of an improvement. 

- The section is about "when should I use this" and less about implementation details, which are documented below. 
- "with the information" feels a bit off sentence wise and doesn't refer back to something what that information is
- "using either the `author` and `license`" — either is not correct here, it's not one or the other, they are both used for different things.